lib/issue_badge_settings_controller_patch.rb

```diff
@@ -0,0 +1,43 @@
+# Patch for banner plugin. This affects in "plugin" action of Redmine Settings
+# controller.
+# Now banner plugin does not have own model(table). So, datetime informations
+# are stored as string and required datetime validation by controller.
+#
+# TODO Store banner settings to banner's own model (table).
+#
+module IssueBadgeSettingsControllerPatch
+  unloadable
+
+  def self.included(base)
+    base.send(:include, ClassMethods)
+    base.class_eval do
+      alias_method_chain(:plugin, :issue_badge)
+    end
+  end
+
+  module ClassMethods
+    def plugin_with_issue_badge
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for plugin_with_issue_badge is too high. [18.68/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [14/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)